src/jdk.httpserver/share/classes/module-info.java

```diff
@@ -37,6 +37,60 @@
+ * <ul>
+ * <li><p><b>{@systemProperty sun.net.httpserver.idleInterval}</b> (default: 30 sec)<br>
+ * Maximum duration in seconds which an idle connection is kept open.
+ * </li>
+ * <li><p><b>{@systemProperty sun.net.httpserver.clockTick}</b> (default: 10 sec)<br>
+ * The granularity of the idle interval timer in seconds. The default value might need to
+ * be reduced if the idle interval itself is reduced.
+ * </li>
+ * <li><p><b>{@systemProperty jdk.httpserver.maxConnections}</b> (default: -1)<br>
+ * The maximum number of open connections at a time. This includes active and idle connections.
+ * If zero or negative, then no limit is enforced.
+ * </li>
+ * <li><p><b>{@systemProperty sun.net.httpserver.maxIdleConnections}</b> (default: 200)<br>
+ * The maximum number of idle connections at a time.
+ * </li>
+ * <li><p><b>{@systemProperty sun.net.httpserver.drainAmount}</b> (default: 64K)<br>
+ * The maximum number of bytes that will be automatically read and discarded from a
+ * request body that has not been completely consumed by its
+ * {@link com.sun.net.httpserver.HttpHandler HttpHandler}. If the number of remaining
+ * unread bytes are less than this limit then the connection will be put in the idle connection
+ * cache. If not, then it will be closed.
+ * </li>
+ * <li><p><b>{@systemProperty sun.net.httpserver.maxReqHeaders}</b> (default: 200)<br>
+ * The maxiumum number of headers accepted in a request. If this limit is exceeded
+ * while the headers are being read, then the connection is terminated and the request ignored.
+ * </li>
+ * <li><p><b>{@systemProperty sun.net.httpserver.maxReqTime}</b> (default: -1)<br>
+ * The maximum duration in milliseconds of a request. In practice, this is a function of
+ * request size, network speed, and handler processing delays. A value less than or equal
+ * to zero means the duration is not limited. If the limit is
+ * exceeded then the connection is terminated and the handler will receive a
+ * {@link java.io.IOException}.
+ * </li>
+ * <li><p><b>{@systemProperty sun.net.httpserver.maxRspTime}</b> (default: -1)<br>
+ * The maximum duration in milliseconds of a response. In practice, this is a function of
+ * response size, network speed, and handler processing delays. A value less than or equal
+ * to zero means the duration is not limited. If the limit is
+ * exceeded then the connection is terminated and the handler will receive a
+ * {@link java.io.IOException}.
+ * </li>
+ * <li><p><b>{@systemProperty sun.net.httpserver.timerMillis}</b> (default: 1 sec)<br>
+ * The granularity of the timer that manages request and response times.
+ * </li>
+ * <li><p><b>{@systemProperty sun.net.httpserver.debug}</b> (default: false)<br>
+ * Boolean value, which if true, generates debug information on the console.
+ * </li>
+ * <li><p><b>{@systemProperty sun.net.httpserver.nodelay}</b> (default: false)<br>
+ * Boolean value, which if true, sets the TCP_NODELAY socket option on all incoming connections.
```

Comment:
Maybe link it to TCP_NODELAY socket option here? `{@link java.net.SocketOptions#TCP_NODELAY}`

Reply:
> {@link java.net.SocketOptions#TCP_NODELAY}
or {@link java.net.StandardSocketOptions#TCP_NODELAY} - it has better docs